spec/std/file_utils_spec.cr

```diff
@@ -97,138 +97,102 @@ describe "FileUtils" do
 
   describe "touch" do
     it "creates file if it doesn't exists" do
-      filename = File.join(__DIR__, "data/test_touch.txt")
-      begin
-        File.exists?(filename).should be_false
-        FileUtils.touch(filename)
-        File.exists?(filename).should be_true
-      ensure
-        File.delete filename
+      with_tempfile("file_utils-touch.txt") do |path|
+        FileUtils.touch(path)
+        File.exists?(path).should be_true
       end
     end
 
-    it "creates multiple files if they doesn't exists" do
-      paths = [
-        File.join(__DIR__, "data/test_touch_1.txt"),
-        File.join(__DIR__, "data/test_touch_2.txt"),
-        File.join(__DIR__, "data/test_touch_3.txt"),
-      ]
-      begin
+    it "creates multiple files if they don't exists" do
+      with_tempfile("file_utils-touch") do |path_prefix|
```

Comment:
might be a problem here: you'll have 3 files created that don't get cleaned after `with_tempfile`, maybe use sth like:
```
with_tempfile("touch_1.txt", "touch_2.txt", "touch_3.txt") do |*paths|
  # ...
end
```
?

Reply:
true, that needs to be refactored.